### PR TITLE
addition to outreach team

### PR DESCRIPTION
**Pull request by FranBonath**
adding myself and Xesus to outreach, moved Renuka to Alumni

### CODE REVIEW

**Comment by FranBonath** [comment]
@ewels there seem to be some broken links that I didn't touch? Can this PR be merged?

File: markdown/about.md [hunk 1]
```diff
@@ -30,13 +30,15 @@ the nf-core alumni hall of fame:
...
+- [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
 
 ## Outreach team {#outreach}
 
 The outreach team is responsible for organising hackathons, the `#bytesize` seminar series and other community outreach efforts.
 This team includes all of the `@core-team` mentioned above and also:
 
-- [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
+- [@FranBonath](https://github.com/FranBonath): Franziska Bonath
+- Xesús M. Abalo
```
Comment (ewels) [inline markdown/about.md]: Let's wait until he gets registered on GitHub :octocat: 👍🏻

File: markdown/about.md [hunk 2]
```diff
@@ -30,13 +30,15 @@ the nf-core alumni hall of fame:
 - [@KevinMenden](https://github.com/KevinMenden): Kevin Menden
 - [@olgabot](https://github.com/olgabot): Olga Botvinnik
 - [@sven1103](https://github.com/sven1103): Sven F.
+- [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
```
Comment (ewels) [inline markdown/about.md:33]: Technically not core team. Can you perhaps just adjust the intro text to be less specific?

```markdown
The following people have earned their place in the nf-core alumni hall of fame:
```

File: markdown/about.md [hunk 3]
```diff
@@ -30,13 +30,15 @@ the nf-core alumni hall of fame:
...
 
 ## Outreach team {#outreach}
 
 The outreach team is responsible for organising hackathons, the `#bytesize` seminar series and other community outreach efforts.
 This team includes all of the `@core-team` mentioned above and also:
 
-- [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
+- [@FranBonath](https://github.com/FranBonath): Franziska Bonath
+- [@Xesus-Abalo](https://github.com/Xesus-Abalo) Xesús M. Abalo
```
Comment (ewels) [inline markdown/about.md]: 🤓 
```suggestion
- [@Xesus-Abalo](https://github.com/Xesus-Abalo): Xesús M. Abalo
```

**Review by ewels — CHANGES_REQUESTED** [review]
Minor, nearly there!

**Review by ewels — APPROVED** [review]
Crocodile Hunter Yes

**Comment by ewels** [comment]
oh, wait - haha. Wondered why I couldn't merge then realised that this PR isn't against the @nf-core fork 😅  You're just asking to merge against your own `master` branch..

There should be an edit button at the top where you can change the target branch, maybe you can also change the repot that it's pointing to? If not, you'll want to close this (without merging) and then open up a new PR against nf-core/nf-co.re (not FranBonath/nf-co.re).

You might need to click this link: 
image

To be able to see this dropdown for the base repo:

image

**Comment by FranBonath** [comment]
PR to wrong depo

**Pull request closed without merging**